podcasts/episodes.go

```diff
@@ -146,6 +147,6 @@ func SearchEpisode(episodes []*Episode, query string) []*Episode {
 
 type ByDate []*Episode
 
-func (a ByDate) Len() int           { return len(a) }
-func (a ByDate) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
-func (a ByDate) Less(i, j int) bool { return a[i].ReleaseDate.After(a[j].ReleaseDate) }
+func (b ByDate) Len() int           { return len(b) }
+func (b ByDate) Swap(i, j int)      { b[i], b[j] = b[j], b[i] }
+func (b ByDate) Less(i, j int) bool { return b[i].ReleaseDate.After(b[j].ReleaseDate) }
```

Comment:
I don't understand why you changed the variable name

Reply:
(b ByDate) seems more consistent with other naming conventions in the code (e *Episode)